sksparse/cholmod.pyx

```diff
@@ -138,6 +140,37 @@ cdef class Factor
 class CholmodError(Exception):
     pass
 
+class CholmodNotPositiveDefiniteError(CholmodError):
+    def __init__(self, message, column=None, factor=None):
+        super().__init__(message)
+        self.column = column
+        self.factor = factor
+
+class CholmodNotInstalledError(CholmodError):
+    def __init__(self, message=""):
+        message = "method not installed: {}".format(message)
+        super().__init__(message)
+
+class CholmoOutOfMemoryError(CholmodError):
```

Comment:
Typo in the classname.
What message does cholmod emit by default?  Does it not already include the nature of the error?
Would be nice to test at least for CholmodInvalidError (not sure the others are easily testable).